library/src/main/java/com/cube/storm/ui/quiz/view/holder/grid/QuizGridItemViewHolder.java

```diff
@@ -57,7 +57,7 @@ public QuizGridItemViewHolder(View view)
 
 		if (badgeProperty != null)
 		{
-			image.populate(badgeProperty.getIcon(), progress);
+			image.populate(badgeProperty.getIcon(), null, progress);
```

Comment:
It was: 
populate(@Nullable final ArrayList<ImageProperty> image, @Nullable final ProgressBar progress)

and you change it to 
 populate(@Nullable final ArrayList<ImageProperty> image, @Nullable final ProgressBar progress, @Nullable final ImageLoadingListener listener)

I think it should be:
i**mage.populate(badgeProperty.getIcon(), progress , null);** or since the variable is null, leave it as it was.